README.md

```diff
@@ -5,13 +5,6 @@
 
 This is the backend for the Simple app to help track hypertensive patients across a population.
 
-## Table of Contents
-
-* [Development](#development)
-* [Documentation](#documentation)
-* [Deployment](#deployment)
-* [Contributing](#contributing)
-
```

Comment:
GitHub has this built in in the markdown viewer, and this ToC is woefully out of date.